scripts/testPrebidDemo.sh

```diff
@@ -16,5 +16,10 @@ echo -e "✅ PREBID TESTLOG: $@"
 echoX "clean previous build"
 ./gradlew clean
 
+#building
+echoX "start building UI tests"
+./gradlew assembleSourceCodeDebugAndroidTest
+
+# testing
 echoX "start UI tests"
```

Comment:
Commenting this line means Android UI Tests will not run in Travis.
We have tried fixing the UI Tests why not run them in Travis????